src/gui/QvisHostProfileWindow.C

```diff
@@ -81,6 +80,8 @@
 #include <ViewerProxy.h>
 #include <InstallationFunctions.h>
 
+#include <rapidjson/document.h>
```

Comment:
Where does rapidjson come from. I assume from something in third_party_builtin.

Reply:
Yes. third_party_builtin. MFEM and Mili readers already use it.